legacy/public.styl

```diff
@@ -10114,6 +10129,11 @@ section.contact-departments .contact-wrapper:hover .contact-metadata,
     .filter-wrapper .drawer {
         position: absolute
     }
+    .views-exposed-topbar .view-filters .dr-c,
+      .filter-wrapper .dr-c {
+          // du ticket #973
+          position: initial
```

Comment:
I've had cross browser issues with "initial" in the past, not sure if that's still the case: https://stackoverflow.com/questions/15343487/is-position-initial-cross-browser-css 

Have you tested in other browsers?